cpp/src/arrow/filesystem/gcsfs_test.cc

```diff
@@ -105,9 +106,21 @@ class GcsTestbench : public ::testing::Environment {
         error += " (exe not found)";
         continue;
       }
-
-      server_process_ = bp::child(exe_path, "-m", "testbench", "--port", port_, group_);
-      if (server_process_.valid() && server_process_.running()) break;
+      // TODO: should I be worried about lifetime of this variable?
+      bp::ipstream output;
```

Comment:
This seems like it should be a member variable, yes

Reply:
When I made it a member variable, the server would become unresponsive halfway through the tests. Possibly some buffer filling up and refusing new input? Unclear.

But leaving as is seems to not have any visible negative consequences when I run the tests.